osrf/mbzirc
# Add all item models to the coast world

Added various small and large item objects on the vessels.

~~I'm opening this PR for visibility first. There is still one small item model to be created. Once that's done, I'll swap out the small items on one of the vessels with the new one.~~ **Update** All items are available on Fuel. This is ready for review

I noticed a drop in RTF from 9x% to 5x% after including all 28 item models into the world (7 vessels with 4 items each). So I added detachable joints between the item models to reduce cost of collision checking. Now I'm getting 8x%~9x% RTF. The joints for the item models on top of the target vessel are detached once the target vessel is identified.

Other notes: 
* Creating a detachable joint between the items and the vessel (with child model being the vessel) seems to cause the surface or hydrodynamics plugin to apply the effect to the incorrect body causing weird vessel motion. So item models are not attached to vessel.
* Each link seems to be able to hold one 1 detachable joint. So the items on a vessel chained together, i.e. small object <-> small object duplicate <-> large object <-> large object duplicate


To test:

1. launch simple_demo world and try moving any item models on Vessel A. You should see that 4 item models are attached together
1. Publish a detach message and try again:

    ```
    ign topic -t "/Vessel_A/detach" -m ignition.msgs.Empty -p "unused: true
    ```

1. launch coast world and check all item models are correctly placed on top of the vessels.
